src/main/java/io/pinecone/clients/Pinecone.java

```diff
@@ -871,6 +871,18 @@ public AsyncIndex getAsyncIndexConnection(String indexName) throws PineconeValid
         return new AsyncIndex(connection, indexName);
     }
 
+    /**
+     * A method to create and return a new instance of the {@link Inference} client.
```

Comment:
It's still the Pinecone client, right?

For the other clients, the gateway to the Inference API is a method on the Pinecone client, so something like `pinecone.inference.embed`.

Reply:
Yes, it is pinecone client which has getInferenceClient().